src/main/resources/application.properties

```diff
@@ -22,10 +22,10 @@ spring.data.mongodb.uri=mongodb://\${MONGO_USERNAME}:\${MONGO_PASSWORD}@\${MONGO
 spring.data.mongodb.database=\${MONGO_DB_NAME}
 
 # Expiration queue
-cdc-queue.storageConnectionString=\${CDC_QUEUE_CONNECTION_STRING}
-cdc-queue.storageQueueName=\${CDC_QUEUE_NAME}
-cdc-queue.ttlSeconds=\${CDC_QUEUE_TTL_SECONDS}
-cdc-queue.timeoutWalletExpired=\${CDC_QUEUE_VISIBILITY_TIMEOUT_SECONDS}
+cdc.queue.storageConnectionString=\${CDC_QUEUE_CONNECTION_STRING}
+cdc.queue.storageQueueName=\${CDC_QUEUE_NAME}
+cdc.queue.ttlSeconds=\${CDC_QUEUE_TTL_SECONDS}
+cdc.queue.timeoutWalletExpired=\${CDC_QUEUE_VISIBILITY_TIMEOUT_SECONDS}
```

Comment:
Don't forget about this timeoutWalletExpired

Reply:
done here d03444b44f9002d141b41c80d2bcda08b08f57c8